dbt_subprojects/dex/models/trades/base/platforms/curvefi_base_base_trades.sql

```diff
@@ -0,0 +1,50 @@
+{{ config(
+    schema = 'curvefi_base',
+    alias = 'base_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+) }}
+
+WITH token_swaps AS (
+    SELECT
+        evt_block_number AS block_number,
+        CAST(evt_block_time AS timestamp(3) with time zone) AS block_time,
+        evt_tx_from AS maker,
+        evt_tx_to AS taker,
+        tokens_sold AS token_sold_amount_raw,
+        tokens_bought AS token_bought_amount_raw,
+        CAST(sold_id AS varbinary) AS token_sold_address,
+        CAST(bought_id AS varbinary) AS token_bought_address,
+        contract_address AS project_contract_address,
+        evt_tx_hash AS tx_hash,
+        evt_index AS evt_index
+    FROM
+        {{ source('curvefi_base', 'StableSwap_evt_TokenExchange') }}
+    {% if is_incremental() %}
+    WHERE
+        {{ incremental_predicate('evt_block_time') }}
+    {% endif %}
```

Comment:
this doesn't fully follow the current logic we have for Curve on other chains.
Is the deployment on base different then on other chains?

Reply:
I am thinking this too.
This implementation covers `stable` pool type swaps.